### PR TITLE
Migrate Openledger sign up functionality Part 1.

### PR DESCRIPTION
**Pull request by MikhailKavalenka**
**Description.**
Registration model selection page on the separate route /registration.

screen shot 2018-10-08 at 1 52 14 pm

Issue #1953

### CODE REVIEW

**Comment by sschiessl-bcp** [comment]
Please link to the issue in the PR, as it's supposed to close it. 

I think this one belongs to #1953

File: app/assets/images-loader.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+require("file-loader?name=model-type-images/[name].svg!./model-type-images/account-active.svg");
```
Comment (sschiessl-bcp) [inline app/assets/images-loader.js]: Something is breaking with the way you are loading and using the images.

If I browse to "/registration" it looks ok. When I browse to "/registration/wallet" and refresh, the images do not load
image

File: app/assets/stylesheets/layout/_all.scss [hunk 2]
```diff
@@ -1 +1,2 @@
 @import "page_layout";
+@import "new_layout";
```
Comment (sschiessl-bcp) [inline app/assets/stylesheets/layout/_all.scss]: Please use meaningful variable names

File: package.json [hunk 3]
```diff
@@ -253,7 +253,7 @@
         "svgo-loader": "^2.1.0",
         "uglifyjs-webpack-plugin": "^1.2.7",
         "url-loader": "^1.0.1",
-        "webpack": "^4.16.5",
+        "webpack": "4.16.5",
```
Comment (sschiessl-bcp) [inline package.json]: Do you see any compatibility issues in future version? Or what is the reasoning to enforce exact version here?

File: app/assets/stylesheets/themes/_light-theme.scss [hunk 4]
```diff
@@ -56,6 +56,12 @@ $pulsate-red-end: #e87f84;
 
 $modal-bg-color: #dadada;
 
+$new-positive-color: #22d173;
```
Comment (startailcoon) [inline app/assets/stylesheets/themes/_light-theme.scss]: Please use meaningful variable names here as well. If it's an alternative to the commonly used positive color, call it `positive-color-alternative`

File: package.json (hunk 3 above)
Comment (MikhailKavalenka) [reply package.json]: Yes, we experienced a problem trying to deploy with the webpack 4.20. It seems like the webpack-cli should be updated to the 3.1.1 version as well, let me just remove this changes.

File: app/assets/images-loader.js (hunk 1 above)
Comment (MikhailKavalenka) [reply app/assets/images-loader.js]: Urls registration/wallet and registration/account will lead to another pages, so the problem won't be actual.

File: app/assets/stylesheets/themes/_light-theme.scss (hunk 4 above)
Comment (MikhailKavalenka) [reply app/assets/stylesheets/themes/_light-theme.scss]: Will do, thanks.

File: app/assets/stylesheets/layout/_all.scss (hunk 2 above)
Comment (MikhailKavalenka) [reply app/assets/stylesheets/layout/_all.scss]: Will do, thanks.

File: app/assets/images-loader.js (hunk 1 above)
Comment (sschiessl-bcp) [reply app/assets/images-loader.js]: I'm curious as to how this is emerging. Have you found a reason?

**Pull request merged**